todo/src/components/TodoItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { changeTodo } from '../actions';
+
+class TodoItem extends React.Component {
+  state = {
```

Comment:
I'm curious about why you chose to make this into a stateful class component, seems like you are duplicating a lot of effort here.